### ISSUE TITLE
No possibility to enter _LOC:NAME:ABBR:TYPE

### ISSUE THREAD

**Issue by Jefferson49** [vesta-webtrees-2-custom-modules/vesta_shared_places#145]
Observed with vesta 2.1.15.0.3 and webtrees 2.1.15

There seems to be no possibility to enter data for _LOC:NAME:ABBR:TYPE. even if "Edit with all GEDCOM tags" is chosen.

![_LOC ABBR TYPE](https://user-images.githubusercontent.com/81484983/212534332-4a1d4c05-d1e4-4cf9-ba8c-1410e5ca30fa.JPG)

From the Addendum specification, it should be available:
```
0 @<XREF:_LOC>@ _LOC {1:1}
1 NAME <PLACE_NAME> {1:M}
2 DATE <DATE_VALUE> {0:1}
2 ABBR <ABBREVIATION_OF_NAME> {0:M}
3 TYPE <TYPE_OF_ABBREVIATION> {0:1}
```

For existing data, the abbreviation is shown correctly:

![_LOC ABBR TYPE existing](https://user-images.githubusercontent.com/81484983/212534448-ef161374-80a7-4ad5-94e1-cfb5ecd42d96.JPG)

I do not know if this is missing in the GEDCOM-L tag customization in webtrees, or if it is missing in vesta shared places.

**Comment by ric2016** [comment]
It's missing in webtrees, [here](https://github.com/fisharebest/webtrees/blob/6f5bd0bfab12b325068b1a1c83e0c767fb752964/app/CustomTags/GedcomL.php#L261):

That line should be
`'_LOC:NAME:ABBR'                  => new CustomElement(I18N::translate('Abbreviation'), ['TYPE' => '0:1']),`

**Comment by Jefferson49** [comment]
ok, thank You very much for analyzing. I will file an issue in webtrees

**Comment by ric2016** [comment]
I see this has been handled already in webtrees (https://github.com/fisharebest/webtrees/pull/4725), therefore I'll close here.

**Comment by Jefferson49** [comment]
yes, instead of opening an issue, I directly started a pull request with the proposed code, and Greg already accepted :-)
